cmd/common/trace/tcpconnect.go

```diff
@@ -0,0 +1,118 @@
+// Copyright 2022 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package trace
+
+import (
+	"fmt"
+	"strings"
+
+	commonutils "github.com/kinvolk/inspektor-gadget/cmd/common/utils"
+	tcpConnectTypes "github.com/kinvolk/inspektor-gadget/pkg/gadgets/trace/tcpconnect/types"
+	"github.com/spf13/cobra"
```

Comment:
nit: we're trying to organize the imports paths in the following order:
- stdlib
- external deps
- internal packages 

```suggestion
	"fmt"
	"strings"

	"github.com/spf13/cobra"
	
	commonutils "github.com/kinvolk/inspektor-gadget/cmd/common/utils"
	tcpConnectTypes "github.com/kinvolk/inspektor-gadget/pkg/gadgets/trace/tcpconnect/types"
```

Reply:
updated!